logbook-servlet/src/main/java/org/zalando/logbook/servlet/LogbookFilter.java

```diff
@@ -83,7 +83,13 @@ public void doFilter(final HttpServletRequest httpRequest, final HttpServletResp
     private void write(RemoteRequest request, LocalResponse response, ResponseWritingStage writing) throws IOException {
         final AtomicBoolean attribute = (AtomicBoolean) request.getAttribute(responseWritingStageSynchronizationName);
         if (!attribute.getAndSet(true)) {
-            response.flushBuffer();
+            try {
+                response.flushBuffer();
+            } catch (IOException ignored) {
```

Comment:
One question: Could it happen that we are, in some situations, interested in the underlying IO Exception and we want to react to it in some other way instead of surpressing it?